Source/WebCore/rendering/RenderCombineText.cpp

```diff
@@ -99,7 +100,7 @@ void RenderCombineText::combineTextIfNeeded()
 
     // An ancestor element may trigger us to lay out again, even when we're already combined.
     if (m_isCombined)
-        RenderText::setRenderedText(originalText());
+        RenderText::setRenderedText(m_renderedText.impl());
```

Comment:
```suggestion
        RenderText::setRenderedText(m_renderedText);
```
I realized this should just work.

Reply:
Done.